Draft: Release integration (#120)
recognai/rubrix: docs/getting_started/setup&installation.rst

^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

For this method you first need to install `Docker Compose <https://docs.docker.com/compose/install/>`_.

> Create your desired folder:
> 
> .. code-block:: bash
> 
>    mkdir rubrix && cd rubrix
> 
> Then you can launch the docker-contained webapp with the following command:

Reviewer: ```suggestion
Then, create a folder:

.. code-block:: bash

   mkdir rubrix && cd rubrix

and launch the docker-contained webapp with the following command:
```
